parfive/downloader.py

```diff
@@ -371,13 +386,11 @@ def callback(token, future, main_pb):
         """
         timeout = aiohttp.ClientTimeout(**timeouts)
         try:
-            if 'PROXY' in os.environ:
-                kwargs['proxy'] = os.environ['PROXY']
-            if 'PROXY_AUTH' in os.environ:
-                kwargs['proxy_auth'] = os.environ['PROXY_AUTH']
-            if 'PROXY_HEADERS' in os.environ:
-                kwargs['proxy_headers'] = os.environ['PROXY_HEADERS']
-
+            if 'HTTP_PROXY' in os.environ:
```

Comment:
Which one of these we use should be determined by the protocol of the URL.

Reply:
But I can only access the URL after I know the name with which is stored in the env variable, right?

Reply:
I mean the URL of the file we want to download, not the URL of the proxy.

Reply:
I don't think we have a choice really. These are env variable names. The user will set them.
In fact, we are simply passing the proxy URL. The env variable name does not set the protocol

Reply:
What I mean is, if I ask to download `https://mysite/myfile` then we should use the `HTTPS_PROXY` var and if I ask to download `http://othersite/otherfile` then we should use `HTTP_PROXY`.

To do this we can just do this: https://github.com/Cadair/parfive/blob/master/parfive/downloader.py#L154L156